framework/db/QueryBuilder.php

```diff
@@ -65,7 +65,22 @@ class QueryBuilder extends \yii\base\Object
         'EXISTS' => 'buildExistsCondition',
         'NOT EXISTS' => 'buildExistsCondition',
     ];
-
+    /**
+     * @var array map of chars to their replacements in LIKE conditions.
+     * By default it's configured to escape `%`, `_` and `\` with `\`.
+     * @since 2.0.12.
+     */
+    protected $likeEscapingCharacterReplacements = [
+        '%' => '\%',
+        '_' => '\_',
+        '\\' => '\\\\',
+    ];
+    /**
+     * @var string|null character used to escape special characters in LIKE conditions.
+     * By default it's assumed to be `\`.
+     * @since 2.0.12
```

Comment:
A good thing about `'\\'` is that I can simplify tests and get rid of regex magic (and I consider this is good :).